AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2352,7 +2392,16 @@ - (ASLayout *)calculateLayoutThatFits:(ASSizeRange)constrainedSize
     ASEnvironmentStatePropagateDown(layoutSpec, self.environmentTraitCollection);
```

Comment:
@shamanskyh it might be interesting to locally test this on a slow device, to see if it makes sense to include this in the OptionLayoutSpec measurement.  It makes more sense there than as the Generation step, and including it may be helpful as it would allow us to detect regressions in the environment system + overall account for more of the total work going on in these key methods.